network/network.go

```diff
@@ -86,7 +114,30 @@ func getSubnetsClient() network.SubnetsClient {
 }
 
 // CreateVirtualNetworkSubnet creates a subnet
-func CreateVirtualNetworkSubnet() {}
+func CreateVirtualNetworkSubnet(ctx context.Context, vnetName, subnetName string) (subnet network.Subnet, err error) {
+	subnetsClient := getSubnetsClient()
+
+	future, err := subnetsClient.CreateOrUpdate(
+		ctx,
+		helpers.ResourceGroupName(),
+		vnetName,
```

Comment:
does the vnet already needs to exist when creating this ?

Reply:
Yes